dataflow/gpu-examples/tensorflow-landsat/Dockerfile

```diff
@@ -16,7 +16,7 @@
 #   https://www.tensorflow.org/install/source#gpu
 # For a list of all the nvidia images:
 #   https://ngc.nvidia.com/catalog/containers/nvidia:cuda/tags
-FROM nvcr.io/nvidia/cuda:11.4.2-cudnn8-runtime-ubuntu20.04
+FROM nvcr.io/nvidia/cuda:11.2.2-cudnn8-runtime-ubuntu20.04
```

Comment:
As per https://www.tensorflow.org/install/source#gpu for TensorFlow 2.7 we **must** use CUDA 11.2, not the latest. Otherwise it fails.

Reply:
Can you please add this as an inline comment so that other maintainers and I don't upgrade it by accident?

Reply:
Should we consider pinning this in renovate?